src/attached_probe.cpp

```diff
@@ -478,9 +479,16 @@ bool AttachedProbe::resolve_offset_uprobe(bool safe_mode)
     sym.name = symbol;
     bcc_elf_foreach_sym(probe_.path.c_str(), sym_name_cb, &option, &sym);
 
-    if (!sym.start)
-      throw FatalUserException("Could not resolve symbol: " + probe_.path +
-                               ":" + symbol);
+    if (!sym.start) {
+      const std::string msg = "Could not resolve symbol: " + probe_.path + ":" +
+                              symbol;
+      if (error_on_missing_symbol) {
+        throw FatalUserException(msg);
+      } else {
+        LOG(WARNING) << msg << ", skipping.";
```

Comment:
I think it'd be more clear to say "skipping probe." rather than just "skipping."

Reply:
Good point, I updated the error/warning messages.